### DISCUSSION TITLE
Song info in window title not updated

### DISCUSSION THREAD

**Discussion by ghmon** [smplayer-dev/smplayer#183] [General]
Version: 21.1.0 (revision 9506) - Development version

I am using smplayer now since two years, always compiled from the local github repo. When i start a radio stream the song artist/title is shown correctly in the window title of smplayer. When the song ends and the next song is started, the window title remains the same for the previous song. When i start the stream playing freshly, the song title is shown again correctly.

Using mpv 0.27.2. Maybe it's a mpv option i am missing?

Thanks for any help.

**Comment by smplayer-dev** [comment]
Can you provide a sample radio stream where this happens?

**Reply by ghmon** [reply]
Btw: mpv http://79.120.77.11:8000/rock?type=.flv

Title is updated on every song change.

**Comment by ghmon** [comment]
> Can you provide a sample radio stream where this happens?

Sure - it happens on every stream, e.g. http://79.120.77.11:8000/rock?type=.flv

**Comment by smplayer-dev** [comment]
I think the problem is fixed in version 9903. Could you test it?

**Reply by ghmon** [reply]
👍 Freshly compiled from the repo - works! Great! Thank you very much.
